src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -12,74 +12,183 @@ import { useSelector } from "hooks/useSelector";
 
 import { Map, RoomModal } from "./components";
 
-import "./PartyMap.scss";
 import AnnouncementMessage from "components/molecules/AnnouncementMessage/AnnouncementMessage";
 import SparkleFairiesPopUp from "components/molecules/SparkleFairiesPopUp/SparkleFairiesPopUp";
 
+import "./PartyMap.scss";
+
 const partyMapVenueSelector = (state: RootState) =>
   state.firestore.ordered.currentVenue?.[0] as PartyMapVenue;
 
 export const PartyMap: React.FC = () => {
-  const [isRoomModalOpen, setIsRoomModalOpen] = useState(false);
+  const currentVenue = useSelector(partyMapVenueSelector);
+
+  // TODO: can we just make this a boolean that is set from isTruthy(selectedRoom)?
+  const [isRoomModalOpen, setRoomModalOpen] = useState(false);
   const [selectedRoom, setSelectedRoom] = useState<
     PartyMapRoomData | undefined
   >();
```

Comment:
I had a very long and in-depth explanation here, but in brief the reason for the two variables is so the hiding animation doesn't happen with no data inside the modal. If the modal only hides when there's no value for `selectedRoom` it will look very strange during the hide animation (blank values where there were titles, contents etc.)